app/services/support_interface/candidate_autofill_usage_export.rb

```diff
@@ -0,0 +1,157 @@
+        {
+          'Field' => 'Other type',
+          'Value entered' => type,
+          'Frequency' => count,
+          'Free text?' => !other_types.include?(type),
+        }
+      end
+    end
+
+    def other_grade_output
+      counts_for(level: :other, attribute: :grade)
+        .map do |grade, count|
+        {
+          'Field' => 'Other grade',
+          'Value entered' => grade,
+          'Frequency' => count,
+          'Free text?' => !other_grades.include?(grade),
+        }
+      end
+    end
+
+    def degree_grade_output
+      counts_for(level: :degree, attribute: :grade)
+        .map do |grade, count|
+        {
+          'Field' => 'Degree grade',
+          'Value entered' => grade,
+          'Frequency' => count,
+          'Free text?' => !degree_grades.include?(grade),
+        }
+      end
+    end
+
+    def degree_institution_output
+      counts_for(level: :degree, attribute: :institution_name)
+        .map do |institution, count|
+        {
+          'Field' => 'Degree institution',
+          'Value entered' => institution,
+          'Frequency' => count,
+          'Free text?' => !degree_institutions.include?(institution),
+        }
+      end
+    end
+
+    def degree_subject_output
+      counts_for(level: :degree, attribute: :subject)
+        .map do |subject, count|
+        {
+          'Field' => 'Degree subject',
+          'Value entered' => subject,
+          'Frequency' => count,
+          'Free text?' => !degree_subjects.include?(subject),
+        }
+      end
+    end
+
+    def degree_type_output
+      counts_for(level: :degree, attribute: :qualification_type)
+        .map do |type, count|
+        {
+          'Field' => 'Degree type',
+          'Value entered' => type,
+          'Frequency' => count,
+          'Free text?' => !degree_types.include?(type),
+        }
+      end
+    end
+
+    def counts_for(level:, attribute:)
+      case level
+      when :degree
+        degree_qualifications
+          .group(attribute)
+          .count
+      when :other
+        other_qualifications
+          .group(attribute)
+          .count
+      end
+    end
+
+    def degree_qualifications
+      @degree_qualifications ||= ApplicationQualification
+        .joins(:application_form)
+        .where(
+          level: 'degree',
+          application_forms: {
+            phase: 'apply_1',
+            recruitment_cycle_year: RecruitmentCycle.current_year,
+          },
+        )
+        .all
+    end
+
+    def other_qualifications
+      @other_qualifications ||= ApplicationQualification
+        .joins(:application_form)
+        .where(
+          level: 'other',
+          application_forms: {
+            phase: 'apply_1',
+            recruitment_cycle_year: RecruitmentCycle.current_year,
+          },
+        )
+        .all
+    end
+
+    def free_text?(grade)
+      !all_possible_grades.include?(grade)
+    end
+
+    def all_possible_grades
```

Comment:
If `free_text?` is not used or required, then I think we can remove this too.

Reply:
I'll remove